code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -0,0 +1,816 @@
+				uses--
+			else
+				to_chat(user, "<span class='warning'>[C] is already bound.</span>")
+		else
+			to_chat(user, "<span class='warning'>You fail to shackle [C].</span>")
+	else
+		to_chat(user, "<span class='warning'>[C] is already bound.</span>")
+
+
+/obj/item/restraints/handcuffs/energy/cult //For the shackling spell
+	name = "shadow shackles"
+	desc = "Shackles that bind the wrists with sinister magic."
+	trashtype = /obj/item/restraints/handcuffs/energy/used
+	flags = DROPDEL
+
+/obj/item/restraints/handcuffs/energy/cult/used/dropped(mob/user)
+	user.visible_message("<span class='danger'>[user]'s shackles shatter in a discharge of dark magic!</span>", \
+							"<span class='userdanger'>Your [src] shatters in a discharge of dark magic!</span>")
+	. = ..()
+
+
+//Construction: Converts 50 metal to a construct shell, plasteel to runed metal, airlock to brittle runed airlock, a borg to a construct, or borg shell to a construct shell
+/obj/item/melee/blood_magic/construction
+	name = "Twisting Aura"
+	desc = "Corrupts certain metalic objects on contact."
+	invocation = "Ethra p'ni dedol!"
+	color = "#000000" // black
+	var/channeling = FALSE
+
+/obj/item/melee/blood_magic/construction/examine(mob/user)
+	. = ..()
+	. += {"<u>A sinister spell used to convert:</u>\n
+	Plasteel into runed metal\n
+	[METAL_TO_CONSTRUCT_SHELL_CONVERSION] metal into a construct shell\n
+	Living cyborgs into constructs after a delay\n
+	Cyborg shells into construct shells\n
+	Airlocks into brittle runed airlocks after a delay (harm intent)"}
+
+/obj/item/melee/blood_magic/construction/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	if(proximity_flag && iscultist(user))
+		if(channeling)
+			to_chat(user, "<span class='cultitalic'>You are already invoking twisted construction!</span>")
+			return
+		var/turf/T = get_turf(target)
+		if(istype(target, /obj/item/stack/sheet/metal))
+			var/obj/item/stack/sheet/candidate = target
+			if(candidate.use(METAL_TO_CONSTRUCT_SHELL_CONVERSION))
+				uses--
+				to_chat(user, "<span class='warning'>A dark cloud emanates from your hand and swirls around the metal, twisting it into a construct shell!</span>")
+				new /obj/structure/constructshell(T)
+				SEND_SOUND(user, sound('sound/magic/cult_spell.ogg', 0, 1, 25))
+			else
+				to_chat(user, "<span class='warning'>You need [METAL_TO_CONSTRUCT_SHELL_CONVERSION] metal to produce a construct shell!</span>")
+				return
+		else if(istype(target, /obj/item/stack/sheet/plasteel))
+			var/obj/item/stack/sheet/plasteel/candidate = target
+			var/quantity = candidate.amount
+			if(candidate.use(quantity))
+				uses --
+				new /obj/item/stack/sheet/runed_metal(T,quantity)
+				to_chat(user, "<span class='warning'>A dark cloud emanates from you hand and swirls around the plasteel, transforming it into runed metal!</span>")
+				SEND_SOUND(user, sound('sound/magic/cult_spell.ogg', 0, 1, 25))
+		else if(istype(target,/mob/living/silicon/robot))
```

Comment:
okay not gonna mark em all but try to put spaving between vars istype(target, /mob/living/silicon/robot) instead of istype(target,/mob/living/silicon/robot)